graph/typeDefs.graphql

```diff
@@ -497,6 +510,9 @@ type RootQuery {
   # Metrics related to user actions are saturated into the assets returned. The
   # sort will affect if it will allow
   metrics(from: Date!, to: Date!, sort: ACTION_TYPE!, limit: Int = 10): [Asset]
+
+  # Users returned based on a query.
+  usersFlagged(query: UsersQuery): [User]
```

Comment:
Hence adding the `action_type` field to the `UsersQuery`, it would function in a similar fashion to the `comments` edge in that manner.